source/brailleDisplayDrivers/albatross/driver.py

```diff
@@ -114,14 +111,11 @@ def __init__(self, port="Auto"):
 		self._exitEvent = Event()
 		# Thread for read
 		self._handleRead = None
-		# Display function is called manually when display is switched back on
-		# or exited from internal menu. Ensuring data integrity.
-		self._displayLock = Lock()

```

Comment:
could you explain a bit why the lock was removed? It's somewhat concerning

Reply:
When reconnected or exited from internal menu braille.handler._displayWithCursor() is executed so there should be no reason to execute display function manually. My assumption is that possible conflicts are thus resolved elsewhere and thus lock is redundant.

Reply:
The display function should be  able to handle being executed from other threads. While I think current contexts may be safe for the driver, it's better to keep the lock.